bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -341,155 +153,48 @@ public Auditor(final String bookieIdentifier,
-                numLedgersSoftlyAdheringToPlacementPolicy);
-
-        numUnderreplicatedLedgersElapsedRecoveryGracePeriod = new Gauge<Integer>() {
-            @Override
-            public Integer getDefaultValue() {
-                return 0;
-            }
-
-            @Override
-            public Integer getSample() {
-                return numOfURLedgersElapsedRecoveryGracePeriodGuageValue.get();
-            }
-        };
-        this.statsLogger.registerGauge(ReplicationStats.NUM_UNDERREPLICATED_LEDGERS_ELAPSED_RECOVERY_GRACE_PERIOD,
-                numUnderreplicatedLedgersElapsedRecoveryGracePeriod);
-
-        numLedgersHavingNoReplicaOfAnEntry = new Gauge<Integer>() {
-            @Override
-            public Integer getDefaultValue() {
-                return 0;
-            }
-
-            @Override
-            public Integer getSample() {
-                return numLedgersHavingNoReplicaOfAnEntryGuageValue.get();
-            }
-        };
-        this.statsLogger.registerGauge(ReplicationStats.NUM_LEDGERS_HAVING_NO_REPLICA_OF_AN_ENTRY,
-                numLedgersHavingNoReplicaOfAnEntry);
-        numLedgersHavingLessThanAQReplicasOfAnEntry = new Gauge<Integer>() {
-            @Override
-            public Integer getDefaultValue() {
-                return 0;
-            }
-
-            @Override
-            public Integer getSample() {
-                return numLedgersHavingLessThanAQReplicasOfAnEntryGuageValue.get();
-            }
-        };
-        this.statsLogger.registerGauge(ReplicationStats.NUM_LEDGERS_HAVING_LESS_THAN_AQ_REPLICAS_OF_AN_ENTRY,
-                numLedgersHavingLessThanAQReplicasOfAnEntry);
-        numLedgersHavingLessThanWQReplicasOfAnEntry = new Gauge<Integer>() {
-            @Override
-            public Integer getDefaultValue() {
-                return 0;
-            }
-
-            @Override
-            public Integer getSample() {
-                return numLedgersHavingLessThanWQReplicasOfAnEntryGuageValue.get();
-            }
-        };
-        this.statsLogger.registerGauge(ReplicationStats.NUM_LEDGERS_HAVING_LESS_THAN_WQ_REPLICAS_OF_AN_ENTRY,
-                numLedgersHavingLessThanWQReplicasOfAnEntry);
-        numUnderReplicatedLedgers = new Gauge<Integer>() {
-            @Override
-            public Integer getDefaultValue() {
-                return 0;
-            }
-
-            @Override
-            public Integer getSample() {
-                return underReplicatedLedgersGuageValue.get();
-            }
-        };
-        this.statsLogger.registerGauge(NUM_UNDER_REPLICATED_LEDGERS_GUAGE, numUnderReplicatedLedgers);
-
         this.bkc = bkc;
         this.ownBkc = ownBkc;
         this.admin = admin;
         this.ownAdmin = ownAdmin;
         initialize(conf, bkc);
 
+        AuditorTask.ShutdownTaskHandler shutdownTaskHandler = this::submitShutdownTask;
+        AuditorTask.SubmitTaskHandler submitBookieCheckTaskHandler = this::submitBookieCheckTask;
```

Comment:
It's only for AuditorBookieCheckTask, not define it in the AuditorTask.

Reply:
I don't think so. If any task needs to do some operations, it can do the operation alone. The SubmitTaskHandler is not common for different tasks.

Reply:
I got it. I have submit your feedback. PTAL. Thanks.